rules/sort-union-types.ts

```diff
@@ -117,69 +118,48 @@ export default createEslintRule<Options, MESSAGE_ID>({
         sourceCode,
       })
 
+      let typeToGroupMap: Record<string, Group> = {
```

Comment:
Same comment here as in `sort-intersection-types`.

Reply:
Back to the previous version.